src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsCaseMapping.java

```diff
@@ -83,40 +83,43 @@ public static boolean isInterpreterRequired(SscsCaseData caseData) {
     }
 
     public static boolean isInterpreterRequiredOtherParties(List<CcdValue<OtherParty>> otherParties) {
-        return nonNull(otherParties) && otherParties.stream().map(CcdValue::getValue).anyMatch(o -> isInterpreterRequiredHearingOptions(o.getHearingOptions()));
+        return nonNull(otherParties) && otherParties.stream().map(CcdValue::getValue)
+            .anyMatch(o -> isInterpreterRequiredHearingOptions(o.getHearingOptions()));
```

Comment:
minor: can you use lambda here?

Reply:
I don't have any idea how to do. Would you mind demonstrate it?
@dattipoeHMCTS Would you mind I change this part of code?